Description: make the build system obey CPPFLAGS too
Author: Mattia Rizzolo <mattia@debian.org>
Forwarded: no
Last-Update: 2016-09-27

--- a/configure
+++ b/configure
@@ -3151,7 +3151,7 @@
 	# XXX: not sure if /usr/local is necessary.
 	# XXX: linux libwrap needs -lnsl. configure should check for
 	#      existence of libnsl instead of hard-coding
-	CPPFLAGS="$CFLAGS -I/usr/local/include"; export CPPFLAGS
+	CPPFLAGS="$CPPFLAGS $CFLAGS -I/usr/local/include"; export CPPFLAGS
 	LDFLAGS="$LDFLAGS -L/usr/local/lib -L/lib"; export LDFLAGS
 	LIBS="-lnsl -lcrypt $LIBS"; export LIBS
         $as_echo "#define LINUX 1" >>confdefs.h
@@ -3160,14 +3160,14 @@
 	# XXX: does linux need glibc: -DGLIBC
     ;;
     *mips* )
-	CPPFLAGS="$CFLAGS -I/usr/local/include"; export CPPFLAGS
+	CPPFLAGS="$CPPFLAGS $CFLAGS -I/usr/local/include"; export CPPFLAGS
 	LDFLAGS="$LDFLAGS -L/usr/local/lib"; export LDFLAGS
 	LIBS="-lcrypt $LIBS"; export LIBS
         $as_echo "#define MIPS 1" >>confdefs.h
 
     ;;
     * )
-	CPPFLAGS="$CFLAGS -I/usr/local/include"; export CPPFLAGS
+	CPPFLAGS="$CPPFLAGS $CFLAGS -I/usr/local/include"; export CPPFLAGS
 	LDFLAGS="$LDFLAGS -L/usr/local/lib"; export LDFLAGS
     ;;
 esac
